modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/Actions.tsx

```diff
@@ -3,64 +3,45 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import {IItemsActions} from '..';
 import {navigate, openConfirmModal} from 'frontend-js-web';
-import PropTypes from 'prop-types';
 import React, {useContext, useState} from 'react';
 
-import FrontendDataSetContext from '../FrontendDataSetContext';
+import FrontendDataSetContext, {
+	IFrontendDataSetContext,
+} from '../FrontendDataSetContext';
 import {ACTION_ITEM_TARGETS} from '../utils/actionItems/constants';
-import {formatActionURL} from '../utils/index';
+import filterItemActions from '../utils/actionItems/filterItemActions';
+import formatActionURL from '../utils/actionItems/formatActionURL';
```

Comment:
At some point, we should reshuffle those utils, to be in the optimal scope (`/actions/utils/*`), and not in global scope. But, it doesn't have to be in this PR.

Reply:
That's the idea, but there are still some utilities in the `utils/index.js` file that need to be extracted and also others that do not have a clear scope.